src/test/java/com/klodnicki/Bike/v2/service/ChargingStationServiceHandlerTest.java

```diff
@@ -0,0 +1,158 @@
+package com.klodnicki.Bike.v2.service;
+
+import com.klodnicki.Bike.v2.DTO.station.StationForAdminResponseDTO;
+import com.klodnicki.Bike.v2.model.entity.Bike;
+import com.klodnicki.Bike.v2.model.entity.ChargingStation;
+import com.klodnicki.Bike.v2.repository.BikeRepository;
+import com.klodnicki.Bike.v2.repository.ChargingStationRepository;
+import org.junit.jupiter.api.Test;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class ChargingStationServiceHandlerTest {
+
+    @Autowired
+    private ChargingStationRepository chargingStationRepository;
+    @Autowired
+    private BikeRepository bikeRepository;
+    @Autowired
+    private ChargingStationServiceHandler chargingStationServiceHandler;
+    @Autowired
+    private ModelMapper modelMapper;
+
+    @Test
+    public void add_ShouldAddToDatabase_WhenGivenCorrectArguments() {
+        //given
+        chargingStationRepository.deleteAll();
```

Comment:
Every test in this class has some similarities. This line:
`chargingStationRepository.deleteAll();` 
could easily be extracted into setUp or tearDown method (marked with `@BeforeEach` or `@AfterEach`)

Reply:
that is absolutely true! I wanted to make that and I forgot! Maybe I'm getting old LOL